x/evm/types/statedb.go

```diff
@@ -745,13 +744,8 @@ func (csdb *CommitStateDB) ForEachStorage(addr ethcmn.Address, cb func(key, valu
 			continue
 		}
 
-		_, content, _, err := rlp.Split(value.Bytes())
-		if err != nil {
-			return err
-		}
```

Comment:
idk why I added this originally (prb to match geth), the value bytes are not RLP encoded. This should fix the issue with the contract deployments